app/src/main/java/com/texthip/thip/ui/navigator/navigations/GroupNavigation.kt

```
@@ -29,11 +78,181 @@ fun NavGraphBuilder.groupNavigation(navController: NavHostController) {
-                navController.navigateBack()
+                navigateBack()
             }
         )
     }
+    
+    // Group Done 화면
+    composable<GroupRoutes.Done> {
+        val parentEntry = remember(navController) {
+            try {
+                navController.getBackStackEntry(MainTabRoutes.Group)
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)